### ISSUE TITLE
A.01.03.iso_19115_19119.model.md

### ISSUE THREAD

**Issue by jensscheerlinck** [inspire-eu-validation/ats-discovery-service#8]
_This issue has been extracted from the issue list on:https://ies-svn.jrc.ec.europa.eu/issues/2685_
# Comment

Title is wrong:
A Unique identification for this test

**Comment by jensscheerlinck** [comment]
# Proposed Resolution

Change title to: A.01.03 ISO 19115/19119 Model

**Comment by PeterParslow** [comment]
Agreed (the title within the text, not the "name" of the test, which is correct.
